Jenkinsfile

```diff
@@ -43,20 +34,11 @@ pipeline {
         ISTIO_DOMAIN   = "${env.stagingDomain}"
       }
       steps {
-        notifyStageStart()
         container('skaffold') {
           unstash 'build'
           sh "skaffold deploy -a image.json -n ${TILLER_NAMESPACE}"
         }
       }
-      post {
-        success {
-          notifyStageEnd([status: "Successfully deployed to staging:\nspringtrader.${env.stagingDomain}/spring-nanotrader-web/"])
```

Comment:
You still need the string after `status`...but now it belongs in a new directive names `stageMessage`.  so should be:

`stageMessage: "Successfully deployed to staging..."`


Put the `stateMessage` in the steps block above